src/pages/T4.js

```diff
@@ -0,0 +1,60 @@
+const { css } = require('emotion')
+const { theme } = require('../styles.js')
+const { html } = require('../utils.js')
+const Layout = require('../components/Layout.js')
+const ErrorList = require('../components/ErrorList.js')
+const LogoutLink = require('../components/LogoutLink.js')
+const SummaryTable = require('../components/SummaryTable.js')
+const ButtonLink = require('../components/ButtonLink.js')
+const ValidationError = require('../components/forms/ValidationError.js')
+const Checkbox = require('../components/forms/Checkbox.js')
+const Button = require('../components/forms/Button.js')
+
+const t4CSS = css`
+  position: relative;
+
+  > div {
+    margin-bottom: ${theme.space.xl};
+  }
+`
+
+const changesButtonCSS = css`
+  width: 200px;
+  background-color: pink;
+`
+
+const consentButtonCSS = css`
+  width: 200px;
+  float: right;
+`
+
+//todo How to get t4 info? Where does it get injected from? user object passed with an array of t4s?
+//is this passed in to the Dashbord object which then passes it to summary table?
+const t4Data = ({ employerName, year, box12, box14, box22 }) => {
+  return [
+    { key: 'Employer Name', value: employerName },
+    { key: 'Year', value: year },
+    { key: 'Box12', value: box12 },
+    { key: 'Box14', value: box14 },
+    { key: 'Box22', value: box22 },
+  ]
+}
+
+const T4 = ({ data = {}, errors = {} }) =>
+  html`
+    <${Layout}>
+      <div class=${t4CSS}>
+        <${LogoutLink} />
+        <h1>Income</h1>
+        <div>
+        <img src='/t4.png' />
```

Comment:
I know it's just a demo image, but can you put an `alt` tag in, maybe "sample T4 form"? I'm trying to keep it as accessible as possible.

I've been running all my pages through the [axe chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US), just in the browser. (There's probably a Firefox equivalent). 

<img width="1186" alt="Screen Shot 2019-04-18 at 3 06 33 PM" src="https://user-images.githubusercontent.com/2454380/56384945-bb5aa700-61eb-11e9-800f-1643a9bd1095.png">